samples/CloudClipboard/CloudClipboard/CloudClipboard/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using OpenTelemetry.Collector.AspNetCore;
+using OpenTelemetry.Collector.Dependencies;
+using OpenTelemetry.Exporter.ApplicationInsights;
+using OpenTelemetry.Stats;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Trace.Sampler;
+
+namespace CloudClipboard
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Enable distributed tracing
+            string instrumentationKey = Environment.GetEnvironmentVariable("AZURE_INSTRUMENTATION_KEY");
```

Comment:
Should there be a fallback to this to app settings?   It seems weird to have two required sources for information.

Reply:
This was added at the last minute.  We could make it more robust.

Reply:
added to use app settings